app/videos/[playbackId]/Comments.js

```diff
@@ -0,0 +1,36 @@
+import { Suspense } from 'react'
+
+async function CommentList({ asset }) {
+  const response = await fetch(
+    `https://reactathon-workshop-functions.vercel.app/api/comments/${asset}`,
+  )
```

Comment:
We can do more I think to show a waterfall instead of just hitting a single route. It's why I added a random `user_id` if we wish to do anything with that.